content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -0,0 +1,320 @@
+
+- **Updated** Kiali addon to version v1.79.0.
+
+- **Added** configurable scaling behavior for Gateway HPA in helm chart
+ ([usage]( https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior)) 
+
+- **Added** `allocateLoadBalancerNodePorts` config option to the Gateway chart.
+  ([Issue #48751](https://github.com/istio/istio/issues/48751))
+
+- **Added** a message to indicate the default webhook shifting from a revisioned installation to a default installation.
+  ([Issue #48643](https://github.com/istio/istio/issues/48643))
+
+- **Added** affinity field to Istiod Deployment. This field is used to control the scheduling of Istiod pods.
+
+- **Added** tolerations field to Istiod Deployment. This field is used to control the scheduling of Istiod pods.
+
+- **Added** support for "profiles" to Helm installation. Try it out with `--set profile=demo`!
+  ([Issue #47838](https://github.com/istio/istio/issues/47838))
+
+- **Added** Set the ztunnel DaemonSet priorityClassName to system-node-critical to ensure it is running on all nodes
+  ([Issue #47867](https://github.com/istio/istio/issues/47867))
+
+- **Fixed** an issue where the webhook generated with `istioctl tag set` is unexpectedly being removed by the installer.
+  ([Issue #47423](https://github.com/istio/istio/issues/47423))
+
+- **Fixed** an issue where uninstalling Istio didn't prune all the resources created by custom files.
+  ([Issue #47960](https://github.com/istio/istio/issues/47960))
+
+- **Fixed** an issue where the installation might be blocked by a "conflicting webhook" error if a previous installation was interrupted or failed.
+  ([Issue #48112](https://github.com/istio/istio/issues/48112))
+
+- **Fixed** an issue where the Istio injection webhook may be modified in dry-run mode.
+  ([Issue #48241](https://github.com/istio/istio/issues/48241))
+
+- **Fixed** an issue `Makefile.k8s.mk` `fetch-root-ca` if-condition not work and can not handle `cacerts` case.
+
+- **Fixed** an issue where injection failed when the name of the pod or its custom owner exceeded 63 characters.
+
+- **Fixed** an issue where the webhook generated with `istioctl tag set` is unexpectedly being removed by the installer.
+  ([Issue #47423](https://github.com/istio/istio/issues/47423))
+
+- **Fixed** an issue causing Istio CNI to stop functioning on minimal/locked down nodes (such as no `sh` binary).
+The new logic runs with no external dependencies, and will attempt to continue if errors are encountered (which could be caused by things like SELinux rules).
+In particular, this fixes running Istio on Bottlerocket nodes.
+  ([Issue #48746](https://github.com/istio/istio/issues/48746))
+
+- **Fixed** Custom injection of the `istio-proxy` container was not working on OpenShift because of the way OpenShift sets pod's `SecurityContext.RunAs` field.
```

Comment:
```suggestion
- **Fixed** custom injection of the `istio-proxy` container was not working on OpenShift because of the way OpenShift sets pods' `SecurityContext.RunAs` field.
```